README.md

```diff
@@ -53,11 +53,11 @@ Create a new python environment and activate it. For example, with `conda`:
 >Note `flow-merge` requires `python>=3.10`
 
 ```bash
-conda env create -n flow-merge
```

Comment:
I tried and it actually works even without the -n switch as it reads the name from the environment.yml. What do you think? Should we leave it out?

Reply:
mm even if you aren't installing from the environment.yml using -f switch?

Reply:
Yes, even without `-f environment.yml` as that's the default according to the reference.

Going without `-n flow-merge` removes the possibility of a typo as the CLI name overrides the one in the environment.yml which is assumed to be correct in this case with higher likelihood.

> If using an environment.yml file (the default), you can name the environment in the first line of the file with 'name: envname' or you can specify the environment name in the CLI command using the -n/--name argument. The name specified in the CLI will override the name specified in the environment.yml file.

> -f, --file
> Environment definition file (default: environment.yml)

https://conda.io/projects/conda/en/latest/commands/env/create.html